Source/ACE.Server/Managers/LandblockManager.cs

```diff
@@ -212,5 +220,47 @@ public static void UnloadLandblocks()
                 }
             }
         }
+
+        public static void Initialize()
```

Comment:
personal preference but I like to see Initialize functions near the top due to their significance. I would probably make it the first function in the class.